docs/program-setup/bare_metal/bare_metal_api.md

```diff
@@ -1,5 +1,33 @@
 # Bare metal APIs
```

Comment:
This page will be deleted, won't it?

Reply:
Yes, but I didn't want to remove it from the branch until we knew we had all the info we needed from it. It's not shown in the docs, only on GitHub.